deployer/src/deployer/search/__init__.py

```diff
@@ -216,6 +216,11 @@ def to_search(file, _index=None):
         # other SPAs like the home page. Skip these.
         return
     doc = data["doc"]
+
+    if doc["mdn_url"].startswith("/en-US/curriculum/"):
+        # Skip curriculum content for now.
+        return
+
```

Comment:
Nit: Would have been better to move this behind line 224 (`locale, slug = doc["mdn_url"].split("/docs/", 1)`), so it works independent of `en-US` (`DEFAULT_LOCALE`).